src/core/os/lnx/lnxGpuMemory.cpp

```diff
@@ -131,7 +131,7 @@ Result GpuMemory::AllocateOrPinMemory(
                 // the pinned memory has special requirement for size and base virtual address.
                 // both supposed to be alinged to page boundary otherwise the pinned down operation
                 // will fail.
-                PAL_ASSERT(m_pPinnedMemory);
+                PAL_ASSERT(m_pPinnedMemory != NULL);
```

Comment:
Please change NULL to nullptr and I'll go ahead and approve.

Reply:
@shadiramadan Thanks! Pushed the changes for both pal and xgl repos.